src/subscription_manager/cert_sorter.py

```diff
@@ -259,8 +260,33 @@ def _scan_entitlement_certs(self):
 
                     product_dict.setdefault(product.id, []).append(ent_cert)
 
+    def get_system_status_id(self):
+        return self.system_status
+
+    @staticmethod
+    def get_status_map():
+        """
+        Get status map
+        :return: status map
+        """
+        # Status map has to be here, because we have to translate strings
+        # when function is called (not during start of application) due to
+        # rhsm.service which can run for very long time
+        status_map = {
+            VALID: _("Current"),
+            PARTIAL: _("Insufficient"),
+            INVALID: _("Invalid"),
+            DISABLED: _("Disabled"),
+            UNKNOWN: _("Unknown")
+        }
+        return status_map
+
     def get_system_status(self):
-        return STATUS_MAP.get(self.system_status, STATUS_MAP['unknown'])
+        """
+        Return string with translated status of system
+        """
```

Comment:
while adding comments is generally a good thing, this was not in the original commit